.gitignore

```diff
@@ -165,3 +165,6 @@ public/locales/**/*.js
 
 deployment_tools_config.json
 
+# betterer cache
+.betterer.cache
```

Comment:
Yes it is. Our long term plan is to store this cache in GCS, because we can see `initialize` step take half a minute or 4 minutes to finish, depending on whether the yarn directory is cached.